src/pages/DragonPage/DragonsSyrup.tsx

```
@@ -100,9 +100,11 @@ const DragonsSyrup: React.FC = () => {
   const sortByEarned = useCallback(
     (a: SyrupInfo, b: SyrupInfo) => {
       const earnedUSDA =
-        Number(a.earnedAmount.toExact()) * Number(a.rewardTokenPriceinUSD ?? 0);
+        Number(a.earnedAmount?.toExact() ?? 0) *
```

Create a prototype extensions method exactOraDefault, so we can hide ?? 0